config/protocol.mainnet.yml

```diff
@@ -50,6 +50,8 @@ ApplicationConfiguration:
   #      DB: 0
   #    BoltDBOptions:
   #      FilePath: "./chains/mainnet.bolt"
+  #    BadgerDbOptions:
+  #      BadgerDir: "./chains/mainnet"
```

Comment:
May we change this to `./chains/mainnet.badger` like how it was done with bolt? This allows to test multiple backends simultaneously and diminish chances of unintentionally corrupting something.

Reply:
Bolt is a bit different in that it's a file, although I'm not sure how LevelDB would react on Badger data and vice versa, they should probably just fail to start correctly. But then again your proposal is probably a safer choice.

Reply:
Checked it: Badger needs a folder, not just a file. In case of `./chains/privnet.badger` it creates folder `privnet.badger` and puts all data into that folder. So, which path should we use?

Reply:
Let's use `${NET}.badger`.